build/build-report.js

```diff
@@ -27,6 +34,23 @@ async function buildStandaloneReport() {
   });
 }
 
+async function buildStandaloneFlowReport() {
```

Comment:
bike shed: should we drop "standalone" from these? I think @connorjclark picked "standalone report" to differentiate the base report from like PSI and the esmodule bundle and not just call it "report". `buildFlowReport` seems sufficient to me, at least

Reply:
agreed. not sure what integration w/ CDT looks like yet, but if need be we can rename it standalone then.